conftest.py

```diff
@@ -0,0 +1,82 @@
+import pytest
+from users.models import User, Team, Role
+from tasks.models import Task, Priority, Status
+
+
+@pytest.fixture
+def teams():
+    """
+    Adds teams to DB
+    """
+    team1 = Team.objects.create(name="Team1",
+                                description="This is a test team")
+    team2 = Team.objects.create(name="Team2",
+                                description="This is a test team")
+    team3 = Team.objects.create(name="Team3",
+                                description="This is a test team")
+    return team1, team2, team3
+
+
+@pytest.fixture
+def users(teams):
+    """
+    Adds users to DB
+    """
+    team1, team2, team3 = teams
+    employees = []
+    managers = []
+    users_counter = 0
+    for i in range(3):
+        for team in (team1, team2):
+            employee = User.create_user(username=f"User{users_counter}",
+                                        email=f"User{users_counter}@redhat.com",
+                                        password="ssSSAD231!@",
+                                        first_name=f"User{users_counter}",
+                                        last_name=f"User{users_counter}",
+                                        role=Role.EMPLOYEE,
+                                        team=team)
+            employees += [employee]
```

Comment:
you can use the append method
```suggestion
            employees.append(employee)
```